packages/next/client/image.tsx

```diff
@@ -438,6 +438,25 @@ export default function Image({
           ) : null}
         </div>
       ) : null}
+      <noscript>
+        <img
```

Comment:
What if the image wasn't lazy? Would we duplicate the image in this scenario (when there's no JS)?

Reply:
Sorry, I'm not sure that I understand exactly what you mean. Are you worried about duplicate network requests?

I don't think there is any possibility that the image will show up twice because there is only one `<img>` inside the `<noscript>` . Duplicate network requests shouldn't be happening either because JavaScript is either enabled (in which case, the browser does not request the `src` of the `img` inside of `<noscript>`) or disabled.

If I've misunderstood the question, could you explain again please (maybe with a small example?)

Thanks!

Reply:
I think @Timer means that when you have both `loading="eager"` and JS disabled, `isVisible` will be true when pre-rendering and the original `img` tag will show the image. This causes 2 images being rendered on the screen.

Reply:
For example you will see 2 images (overlapped) with this:

```jsx
      <Image
        src="..."
        width="20"
        height="20"
        loading="eager"
      />
```

To fix I think we can only add this `<noscript>` tag when `isVisible` is false.

Reply:
Thanks, latest commit only renders the `<noscript>` when `isVisible` is false